pyHS100/cli.py

```diff
@@ -41,11 +44,16 @@ def cli(ctx, ip, host, debug, bulb, plug):
     if ip is not None and host is None:
         host = ip
 
+    if alias is not None and host is None:
+        click.echo("Alias is given, using discovery to find host %s" %
+                   alias)
+        host = find_host_from_alias(alias=alias)
+        click.echo("Found hostname is {}".format(host))
```

Comment:
Could you add a check if the returned host is `None` and return if so, I'd expect that when using `--alias` it should bail out if it cannot discover the device instead of continuing. Otherwise this looks good to be merged :-)